packages/flutter/test/rendering/table_test.dart

```diff
@@ -47,6 +47,20 @@ void main() {
     expect(table.size, equals(const Size(0.0, 0.0)));
   });
 
+  test('Table control test: constrained flex columns', () {
+    final RenderTable table = RenderTable(textDirection: TextDirection.ltr);
+    final List<RenderBox> children =
+        List<RenderBox>.generate(6, (_) => RenderPositionedBox());
```

Comment:
This should just be one line

Reply:
Since you approved the PR, I'm not sure if you are actually wanting me to change this or not. But I can.
However, if I change the children assignment statement into a single line, it will exceed 80 characters (95 total) and dartfmt splits into two lines as I had it.

Reply:
Since you approved the PR, I'm not sure if you are actually wanting me to change this or not. But I can.
However, if I change the children assignment statement into a single line, it will exceed 80 characters (95 total) and dartfmt splits into two lines as I had it.

Reply:
It would be best if you could fix this.  I realize it goes over 80 characters, but our style guide prefers not wrapping this vs going over 80 chars.  I approved it because overall the functionality is good, so once this is fixed it should be good to land.

Reply:
Done.

Reply:
Thanks!